packages/compiler/src/compiler/base/nodes/tags/message.ts

```diff
@@ -120,7 +120,6 @@ function buildMessage<R extends MessageRole>(
 
     return {
       role,
-      id: String(attributes.id),
```

Comment:
not sure what this id is tbh don't see it anywhere in ai sdk documentation

Reply:
It is required in the openai schema. I don't know how vercel would manage this if it is not included when openai requires it...

Reply:
talked offline